src/software/vasp.jl

```diff
@@ -359,14 +359,16 @@ readPROCAR(filename::AbstractString) = open(readPROCAR, filename)
 readPROCAR() = open(readPROCAR, "PROCAR")
 
 """
-    getFermi(io::IO) -> Float64
+    get_fermi(io::IO) -> Float64, Float64
 
-Reads an OUTCAR file and returns the Fermi Energy.
+Reads an OUTCAR file and returns the Fermi Energy and alpha+beta value.
 """
 function get_fermi(io::IO)
     readuntil(io, "E-fermi :")
-    fermi = parse.(Float64, split(readline(io))[1])
-    return fermi
+    ln = split(readline(io))
+    fermi = parse.(Float64, ln[1])
+    alphabeta = parse.(Float64, strip(ln[5],':'))
+    return fermi, alphabeta
```

Comment:
Would it make sense to return a `NamedTuple`, perhaps of the form:
```julia
return (fermi = fermi, alphabeta = alphabeta)
```
Then you could index it with `E[:fermi]` or `E[:alphabeta]`.